make/autoconf/libraries.m4

```diff
@@ -200,20 +200,32 @@ AC_DEFUN_ONCE([LIB_SETUP_MISC_LIBS],
   # Setup posix pthread support
   if test "x$OPENJDK_TARGET_OS" != "xwindows"; then
     LIBPTHREAD="-lpthread"
+  elif test "x$OPENJDK_TARGET_OS" == "xbsd"; then
```

Comment:
This will not work, the != windows will be selected for BSD, and this statement ignored.